packages/core/stories/card/card.doc.stories.mdx

```diff
@@ -26,38 +26,70 @@ This default example is intended to show the way the Card component’s content
   <Story id="core-card--default" />
 </Canvas>
 
-## Interactable
+## Interactable Card
 
-Cards can have interactable styling by setting `interactable={true}`. These cards could be wrapped in Link, for example, for interactivity. We discourage the use of any focusable elements within interactable Cards.
+### Interactable prop
+
+**Deprecated:** Cards can have interactable styling by setting `interactable={true}`. These cards could be wrapped in Link, for example, for interactivity. We discourage the use of any focusable elements within interactable Cards.
+
+### Interactable Card
+
+**Updated:** Cards can have actions by making use of the `InteractableCard` component. We discourage the use of any focusable elements within interactable Cards.
 
 <Canvas>
-  <Story id="core-card--interactable" />
+  <Story id="core-card--interactable-as-link" />
 </Canvas>
 
-## Disabled
+## Disabled Card
+
+### Disabled Card
+
+**Deprecated:** Cards can have disabled styling applied by setting `disabled={true}`.
 
-Cards can have disabled styling applied by setting `disabled={true}`.
+### Disabled (Interactable) Card
+
+**Updated:** Interactable cards can have disabled styling applied by setting `disabled={true}`.
 
 Make sure to as well apply `disabled={true}` to any nested components, such as Text, to ensure disabled styling is also applied to the inner content.
 
 <Canvas>
-  <Story id="core-card--disabled" />
+  <Story id="core-card--interactable-disabled" />
+</Canvas>
+
+## Multiple Cards
+
+Layout components can be used for grouping Card components together.
+
+<Canvas>
+  <Story id="core-card--cards-in-grid-layout" />
 </Canvas>
 
 # API
 
 ```
-import { Card } from "@salt-ds/lab";
+import { Card, InteractableCard } from "@salt-ds/core";
 ```
 
-## Props
+## Card Props
 
 <ArgsTable of={Card} />
 
-## CSS Class
+## Interactable Card Props
+
+<ArgsTable of={InteractableCard} />
+
+## Card CSS Classes
 
 <CSSClassTable of={Card} />
 
-## Characteristics Used
+## Interactable Card CSS Classes
+
+<CSSClassTable of={InteractableCard} />
+
+## Characteristics Used in Card
+
+<CharacteristicUsage of={Card} />
+
+## Characteristics Used in Interactable Card
 
 <CharacteristicUsage of={Card} />
```

Comment:
QQ - is this meant to be  <CharacteristicUsage of={InteractableCard} /> instead?

Reply:
yes, thank you!